### PR TITLE
(chore #54) Improve environment variables documentation

### LINKED ISSUE #54: Improve environment variables documentation.
**Describe the issue**
It would be helpful if the configuration setup for cht-sync and cht-pipeline could be made more understandable.

**Describe the improvement you'd like**
We already have a template for the environment variables that can be used as an example of how to set up an instance. 
I will explain the meaning and purpose of each group of environment variables.

**Context:** This came up because of the confusion when defining values for the environment variables.

### PR DESCRIPTION
**Pull request by lorerod**
closes #54

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,16 @@ CHT Sync has been specifically designed to work in both local development enviro
 - `Docker`
 - An `.env` file containing the environment variable placeholders from the `.env.template` file. The file should be located in the root directory of the project or set by the operating system. The variables should be customized accordingly for the specific deployment needs.
 
+#### Environment variable
+We have four environment variable groups in the `.env.template` file. It is important to understand the difference between them.
```
Comment (andrablaj) [inline README.md:33]:
```suggestion
There are four environment variable groups in the `.env.template` file. To successfully set up `cht-sync`, It is important to understand the difference between them.
```

Comment (andrablaj) [reply README.md:33]: Added a clarification on why it is important to know the difference between the variables, and removed the "we" mention.

File: README.md [hunk 2]
```diff
@@ -29,9 +29,16 @@ CHT Sync has been specifically designed to work in both local development enviro
...
+1. Postgresql and Postgres: Are used to establish the Postgres database to synchronize CouchDB data. It also define the schema and table names to store the CouchDB data. The main objective is to define the environment where the raw CouchDB data is going to be copy to.
```
Comment (andrablaj) [inline README.md:34]:
```suggestion
1. Postgresql and Postgres: Are used to establish the Postgres database to synchronize CouchDB data. They also define the schema and table names to store the CouchDB data. The main objective is to define the environment where the raw CouchDB data will be copied.
```

Comment (andrablaj) [reply README.md:34]: @lorerod I don't see a `Postgresql` variable in the `.env.template`. Where do we use that variable? Am I missing something?

File: README.md [hunk 3]
```diff
@@ -29,9 +29,16 @@ CHT Sync has been specifically designed to work in both local development enviro
...
+2. DBT: These environment variables are exclusive to the DBT configuration. The main objective is to define the environment where the tables and views for the models defined in `CHT_PIPELINE_BRANCH_URL` will be created. It is important to separate this environments from the previous group. `DBT_POSTGRES_USER` and `DBT_POSTGRES_SCHEMA` must be different from `POSTGRES_USER` and `POSTGRES_SCHEMA`. `DBT_POSTGRES_HOST` has to be the Postgres instance created with the environment variables set in the first group.
```
Comment (andrablaj) [inline README.md:35]:
```suggestion
2. DBT: These environment variables are exclusive to the DBT configuration. The main objective is to define the environment where the tables and views for the models defined in `CHT_PIPELINE_BRANCH_URL` will be created. It is important to separate this environment from the previous group. `DBT_POSTGRES_USER` and `DBT_POSTGRES_SCHEMA` must be different from `POSTGRES_USER` and `POSTGRES_SCHEMA`. `DBT_POSTGRES_HOST` has to be the Postgres instance created with the environment variables set in the first group.
```

**Review by andrablaj — CHANGES_REQUESTED** [review]
Thanks for clarifying the doc! I added a few wording comments.

File: README.md (hunk 2 above)
Comment (lorerod) [reply README.md:34]: Both applications use the same set of environment variables.

Comment (andrablaj) [reply README.md:34]: Aaah makes sense, thanks for the explanation!

Comment (andrablaj) [reply README.md:34]: There is this last suggestion to review/solve before I can approve. Can you please look into it, @lorerod ? It's a typo suggestion.

Comment (lorerod) [reply README.md]: Yes! Thank you so much, Andra, for looking into this.

**Pull request merged**